src/DynamoCoreWpf/ViewModels/Core/DynamoViewModelBranding.cs

```diff
@@ -1,22 +1,9 @@
-﻿using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.Text;
-using System.Windows.Media;
-using Dynamo.Wpf.Interfaces;
+﻿using Dynamo.Wpf.Interfaces;
 
 namespace Dynamo.ViewModels
 {
     partial class DynamoViewModel 
     {
         public IBrandingResourceProvider BrandingResourceProvider { get; private set; }
-
-        public ImageSource AboutBoxIcon
-        {
-            get
-            {
-                return BrandingResourceProvider.GetImageSource(ResourceNames.AboutBox.Image);
-            }
-        }
```

Comment:
Now this file looks redundant and `BrandingResourceProvider` can be moved to the main file. Do you see any other chances where we can make use of this file?